solc/CommandLineInterface.cpp

```diff
@@ -542,7 +545,8 @@ Allowed options)",
 			"Switch to linker mode, ignoring all options apart from --libraries "
 			"and modify binaries in place."
 		)
-		(g_argMetadataLiteral.c_str(), "Store referenced sources are literal data in the metadata output.");
+		(g_argMetadataLiteral.c_str(), "Store referenced sources are literal data in the metadata output.")
+		(g_argStandardJSON.c_str(), "Process standard JSON input / output.");
```

Comment:
`Process` seems a bit too technical for me. Also, this disables some other options, does it? Perhaps we should make it more prominent (i.e. put it in the firstline) and say `Switches the compiler into JSON input/output mode.`